src/Illuminate/Support/Testing/Fakes/NotificationFake.php

```diff
@@ -293,7 +299,6 @@ public function send($notifiables, $notification)
      *
      * @param  \Illuminate\Support\Collection|array|mixed  $notifiables
      * @param  mixed  $notification
-     * @param  array|null  $channels
```

Comment:
Can you put this back?

Reply:
whoops…thought I had ignored all of those…sorry! 🤦🏻‍♂️